gelweb/gel2mdt/database_utils/case_handler.py

```diff
@@ -431,7 +544,8 @@ def get_clinician(self):
         if self.case.ir_obj.workspace:
             clinician_details['hospital'] = self.case.ir_obj.workspace[0]
         else:
-            clinician_details['hospital'] = 'unknown'
+            clinician_details['hospital'] = 'Unknown'
```

Comment:
Will need to check the database for all 'unknowns' and rename them to uppercase U